Authorize.Net/Managers/AuthorizeNetMethod.cs

```diff
@@ -264,7 +266,7 @@ public override ProcessPaymentResult ProcessPayment(ProcessPaymentEvaluationCont
                 checkoutform += CreateInput(true, "x_fp_hash", fingerprint);
                 checkoutform += CreateInput(true, "x_currency_code", currency);
                 checkoutform += CreateInput(true, "x_amount", context.Payment.Sum.ToString("F", CultureInfo.InvariantCulture));
-                checkoutform += CreateInput(true, "x_customer_ip", GetIPAddress());
+                checkoutform += CreateInput(true, "x_customer_ip", userIp);
```

Comment:
I think need to add this value only in case `userIp` is not null or empty.

Reply:
done